web/src/blueslip_stacktrace.ts

```diff
@@ -121,7 +121,9 @@ export async function display_stacktrace(ex: Error): Promise<void> {
         .html(
             render_blueslip_stacktrace({
                 error: exception_msg(ex),
-                stackframes,
+                stackframes: stackframes.filter(
+                    (frame) => frame.full_path !== "webpack:///src/blueslip.ts",
```

Comment:
I’m always skeptical of dropping information from developer error messages. For some errors, it might be very important, especially those originating in `blueslip.ts` itself.

Reply:
Hm, OK.  I was on the fence about this one, in that it does remove something that is chaff 99% of the time, but it is certainly true that in the case where it isn't useless, you really need it.